contracts/dao-core/Cargo.toml

```diff
@@ -12,8 +12,11 @@ crate-type = ["cdylib", "rlib"]
 [features]
 # for more explicit tests, cargo test --features=backtraces
 backtraces = ["cosmwasm-std/backtraces"]
+# enables cw20 treasury management
+cw20 = []
 # use library feature to disable all instantiate/execute/query exports
 library = []
+default = ["cw20"]
```

Comment:
So, this means that we will need to release 2 different versions of this contract?

We just need to be sure we are careful not to mix up the CHECKSUMs when we do this, and make this clear.

Reply:
this + your other comment about schema generation are good. my thought is that we can delay working out the specifics of that until we deploy on a chain that doesn't want cw20 support. at that time we can give build instructions for building and verifying without this feature flag enabled.

i've made an issue in rust-optmizer for this so we can see where that goes: https://github.com/CosmWasm/rust-optimizer/issues/107

Reply:
Sounds good, if its too complicated we can always switch to an instantiation param that just blocks access to these flows if it's set.